ofrak_core/src/ofrak/resource.py

```diff
@@ -327,6 +330,7 @@ async def _fetch(self, resource: MutableResourceModel):
         if this resource has been modified.
 
         :raises InvalidStateError: If the local resource model has been modified
+
```

Comment:
```suggestion
```

Is there a good reason to separate these `:raises XXX:` things with new lines? If so, should we be doing it for the other `:raises XXX:` lines added throughout this PR?